### PR TITLE
Trying out Yarn

### PR DESCRIPTION
**Pull request by jorkas**
Seems like a good idea to try it out here, what do you think?

### CODE REVIEW

**Comment by dbackeus** [comment]
Nice 👍

File: gulpfile.js [hunk 1]
```diff
@@ -69,7 +68,7 @@ gulp.task('watch-styleguide', function() {
 
 gulp.task('mnd-bootstrap-sass', function() {
   return gulp.src('src/mnd-bootstrap.scss')
-    .pipe(sass({ outputStyle: 'compact', includePaths: [config.bowerDir]}))
+    .pipe(sass({ outputStyle: 'compact', includePaths: [config.nodeModulesDir]}))
```
Comment (mnd-hound) [inline gulpfile.js]: Line is too long.

File: gulpfile.js [hunk 2]
```diff
@@ -79,16 +78,16 @@ gulp.task('mnd-bootstrap-sass', function() {
...
     .on('error', function (err) { console.log(err.message); })
     .pipe(gulp.dest('public/'))
     .pipe(notify({message: 'Styleguide-sass task complete'}));
 });
 
 gulp.task('styleguide-assets', function() {
   var files = {
-    'bower_components/bootstrap-sass-twbs/assets/javascripts/bootstrap.min.js': 'public/javascripts',
-    'bower_components/jquery/dist/jquery.min.js': 'public/javascripts'
+    './node_modules/bootstrap-sass/assets/javascripts/bootstrap.min.js': 'public/javascripts',
```
Comment (mnd-hound) [inline gulpfile.js]: Line is too long.

**Comment by jorkas** [comment]
[Remove bower from MNDX-Bootstrap](https://trello.com/c/EdqXtctV/1673-remove-bower-from-mndx-bootstrap)

**Comment by bzf** [comment]
Tried it out locally and works great 👍

**Comment by guusli** [comment]
Did a fresh install and all is good 👍

**Pull request merged**